### PR TITLE
Fix to JIRA CDF-880

### PR DESCRIPTION
**Pull request by kleysonr**

### CODE REVIEW

**Comment by pamval** [comment]
@afrjorge @jvelasques please review.

File: core-js/src/main/javascript/cdf/addIns/template/templateBase.js [hunk 1]
```diff
@@ -67,7 +68,7 @@ define([
         type: opt.messages.config.style[type].type || "info",
         icon: opt.messages.config.style[type].icon || "comment"};
       Logger.log(opt.messages.error[message] || "", type);
-      return _.template(opt.messages.config.template)(completeMsg);
+      return _.template(opt.messages.config.template, completeMsg);
```
Comment (jvelasques) [inline core-js/src/main/javascript/cdf/addIns/template/templateBase.js]: underscore lib was recently updated to version 1.8.3, _.template was changed in [1.7.0](http://underscorejs.org/#1.7.0), _.template always returns a function now, so this change will break the functionality

File: core-js/src/main/javascript/cdf/addIns/template/templateBase.js [hunk 2]
```diff
@@ -108,7 +109,7 @@ define([
           switch(opt.templateType.toUpperCase()) {
             case 'UNDERSCORE':
               model = _.defaults({}, model, Utils.propertiesArrayToObject(helpers));
-              html = _.template(Utils.ev(opt.template))(model);
+              html = _.template((_.isFunction(opt.template) ? opt.template() : opt.template), model);
```
Comment (jvelasques) [inline core-js/src/main/javascript/cdf/addIns/template/templateBase.js]: [Utils.ev](https://github.com/webdetails/cdf/blob/master/core-js/src/main/javascript/cdf/dashboard/Utils.js#L375) does this. Again, due to the change to _.template, this change will break

**Comment by jvelasques** [comment]
@kleysonr  the commit message should abide by the [Pentaho Commit Guidelines](http://wiki.pentaho.com/display/PEOpen/Commit+Guidelines)

**Comment by eliofreitas** [comment]
Building pull request

**Comment by brosander** [comment]
http://wingman.pentaho.com:8080/job/wingman/31/

**Comment by brosander** [comment]
Build Commands:
==============
Build:
------
1. mvn -B -fn -DrunITs=true -f core-js/pom.xml clean install && mvn -B -Dpentaho-coding-standards.version=1.0.4 -f core-js/pom.xml site

Cleanup:
-------
mvn -B build-helper:remove-project-artifact



Changed files:
==============
  core-js/src/main/javascript/cdf/addIns/template/templateBase.js

**Comment by afrjorge** [comment]
deprecated by https://github.com/webdetails/cdf/pull/849

**Pull request closed without merging**